### DISCUSSION TITLE
flag provided but not defined : -minerthreads

### DISCUSSION THREAD

**Discussion by sowjanyasegu** [ConsenSys/quorum#1621] [General]
Facing this issue when upgrading from quorum2.6 to quorum22.7.5

flag provided but not defined : -minerthreads

--minerthreads 1 exists in geth command.

**Comment by trinayanbhatt** [comment]
Use the miner flag like this:

`--miner.threads `

**Reply by antonydenyer** [reply]
https://docs.goquorum.consensys.net/deploy/upgrade/migration

**Comment by sowjanyasegu** [comment]
Thanks. Issue resolved
